### ISSUE TITLE
Changing author mid-development

### ISSUE THREAD

**Issue by fpeter8** [Reviewable/Reviewable#721]
Normally only one developer works on our issues, but we have a few edge cases that we cannot really cover right now:
- a developer goes on vacation / is assigned to a different project
- a developer is assigned a production issue, and another developer takes over the draft PR

In these scenarios the original author cannot be changed; we cannot close and reopen the PR because existing comments would be lost.

I don't know enough about the internals of Reviewable to come up with a solution, but maybe a per-commit author track would work (and the commit combining logic could be tweaked here to only work on commits with the same author). At least a "change author on reviewable" button would be appreciated so we could hand PRs over to the new developer.

Possibly related issue: #653

**Comment by pkaminski** [comment]
Yep, this is definitely closely related to #653, but sufficiently distinct to keep a separate issue open.  Tracking authors per commit wouldn't necessarily help, since "author vs reviewer" needs to be review-wide, rather than depending on the current diff bounds (which could vary per file!).  I think the only viable solution is to give optional manual control over this status flag, which would also address #653.
